RedHatInsights/rhsm-subscriptions
# SWATCH-1956: UMB Offering ingestion should write metered y/n flag in Offering table

Jira issue: [SWATCH-1956](https://issues.redhat.com/browse/SWATCH-1956)
Related to [SWATCH-1823](https://issues.redhat.com/browse/SWATCH-1823)
Related to [SWATCH-1840](https://issues.redhat.com/browse/SWATCH-1840)

## Description
- We then have to store a new column on the Offering table. "Metered" column to differentiate between payg and non-payg.
- The product tag determination can be tested using the  /internal/offerings/{sku}/product_tags endpoint. 
- Remove usage of offering.product name.

## Testing
 1. Add any SKU with product name as "RHEL Server" and could have billing_account_id from AWS ex: MCT3557HR - Red Hat Enterprise Linux, Premium (On-Demand, 1 Virtual Node, Hourly, Billing) with billing account id and billing provider.

(SKU addition can be done via endpoint https://mtls.internal.cloud.stage.redhat.com/app/rhsm-capacity-ingress/internal/api/rhsm-subscriptions/v1/internal/subscriptions?reconcileCapacity=true )

from smqe_tools import internal_api_add_sku

internal_api_add_sku(account_number="11490832", sku="MCT3557HR", org_id=app.user.identity.org_id, start_day="2023-10-01", end_day="2023-10-30", billing_provider="aws", billing_account_id="9458475893")

2. Sync capacity.

3. Verify SKU is displayed under rhel-for-x86-eus-payg
